debian/control.in

```diff
@@ -56,3 +57,17 @@ Description: RESTful document oriented database
  standalone node or part of a cluster, and associated other parameters as
  necessary. It adds the systemd / init job and other items needed for a
  system-wide CouchDB instance that is started at boot.
+
+Package: couchdb-nouveau
+Architecture: any
+Depends: systemd,
+         default-jre-headless | java11-runtime-headless | java11-runtime,
+         lsb-base,
+         procps,
+         ${misc:Depends}
+Description: Nouveau is a modern replacement for dreyfus/clouseau
```

Comment:
maybe something high-level front, along the lines of ~“Nouveau adds Lucene-style indexing and searching to Apache CouchDB. …” then the rest?